traffic_ops/app/lib/API/Configs/ApacheTrafficServer.pm

```diff
@@ -76,7 +76,9 @@ sub get_config_metadata {
 		$data_obj->{'info'}->{'cdn_name'}		= $cdn_name;
 		$data_obj->{'info'}->{'cdn_id'}			= $server->cdn->id;
 		$data_obj->{'info'}->{'tm_url'}			= $tm_url;
-		$data_obj->{'info'}->{'tm_cache_url'}	= $tm_cache_url;
+		if ( $tm_cache_url ) {
```

Comment:
should this be called to_cache_url instead? it would be nice to get rid of all references to 'tm' at some point if possible...